tests/unit/src/lib/frontend/components/IdentificationModal.svelte.spec.ts

```diff
@@ -212,3 +212,23 @@ vi.mock('$lib/frontend/endpoints/Endpoints', async () => {
         findUser: vi.fn()
     }
 })
+
+vi.mock('@skeletonlabs/skeleton', async () => {
```

Comment:
Why are we mocking the same module twice?

Reply:
my bad